### ISSUE TITLE
 content_or_style: balanced  # content, style, balanced

### ISSUE THREAD

**Issue by zethfoxster** [derrian-distro/LoRA_Easy_Training_Scripts#238]
does this param exist in easy lora scripts? or can it be added as an extra arg?

**Comment by Jelosus2** [comment]
Do you know where this lies in the code of the sd scripts or do you know any trainer that has it?

**Comment by Jelosus2** [comment]
Ok, so, for what I read in the issues page of the sd scripts repo it seems that the mode "balanced" of the option `content_or_style` (which is in AI-toolkit) is what sd scripts applies by default and then there's an argument called `timestep_sampling` which has a new value called "shift" that shifts the timesteps (which is apparently what AI-toolkit does with the other two options). The latest commit of sd3 branch of sd scripts was pulled 7h ago so make sure to update before you try :)

**Comment by zethfoxster** [comment]
i train flux on the weekends as the trainings always take p the whole day :) ill be sure to give it a try on a fresh pull

**Comment by Jelosus2** [comment]
I'm not sure if it will be added to the UI but you can use it in the extra args section

**Comment by zethfox** [comment]
Alright thanks for the heads up, as long as it works. I know you said what it does above but can you explain in layman term why that would shift the model towards content or style? Do you know?

**Comment by Jelosus2** [comment]
I don't know the math behind but I know is because of the timestep distribution

**Comment by Sorry-Im-Stupid** [comment]
I'm trying to train a style too, I never thought to try timestep. Is there a chart anywhere of where to set the values for that?
